doc-content/enterprise-only/openshift/migration-ocp3-4-postgresql-proc.adoc

```diff
@@ -0,0 +1,113 @@
+----
++
+In this file, find the user name (normally `{PRODUCT_INIT}`), password, and database name (normally `{PRODUCT_INIT}7`) for the database server, for example:
++
+[subs="attributes,verbatim,macros,specialchars,replacements"]
+----
+- name: POSTGRESQL_USER
+  value: {PRODUCT_INIT}
+- name: POSTGRESQL_PASSWORD
+  value: NDaJIV7!
+- name: POSTGRESQL_DATABASE
+  value: {PRODUCT_INIT}7
+----
++
+Record the user name, password, and database name. Do not make any changes to the file.
++
+. To view the service names in the old deployment, run the command:
++
+----
+oc get svc
+----
++
+Find the `postgresql` service that corresponds to the {KIE_SERVER}.
++
+. In a separate terminal window, start port forwarding from the local host to the `postgresql` service, using the name and port number displayed for the service, for example:
++
+----
+oc port-forward service/myapp-postgresql 5432:5432
+----
++
+. Create a dump of the database using the recorded username and database name, for example:
++
+[subs="attributes,verbatim,macros,specialchars,replacements"]
+----
+pg_dump {PRODUCT_INIT}7 -h 127.0.0.1 -U {PRODUCT_INIT} -W >pgdump.sql
+----
++
+When prompted, enter the recorded password. The dump creation can take considerable time.
++ 
+. Stop the port forwarding in the separate window using the `Ctrl`+`C` key combination.
+. Using the `oc` command, log in to the {OPENSHIFT} version 4 infrastructure and switch to the project containing the new deployment.
+. To view the deployment configuration names in the new deployment, run the command:
++
+----
+oc get dc
+----
++
+Find the `postgresql` deployment configuration that corresponds to the {KIE_SERVER}.
++
+. View the configuration YAML of the deployment configuration, for example:
++
+[subs="attributes,verbatim,macros,specialchars,replacements"]
+----
+oc edit dc/myappnew-postgresql
+----
++
+In this file, find the user name (normally `{PRODUCT_INIT}`), password, , and database name (normally `{PRODUCT_INIT}7`) for the database server. Record the user name, password, and database name. Do not make any changes to the file.
++
+. To view the service names in the new deployment, run the command:
++
+----
+oc get svc
+----
++
+Find the `postgresql` service that corresponds to the {KIE_SERVER}.
++
+. In a separate terminal window, start port forwarding from the local host to the `postgresql` service, using the name and port number displayed for the service, for example:
++
+----
+oc port-forward service/myappnew-postgresql 5432:5432
+----
++
+. Restore the database dump using the recorded username and database name, for example:
++
+[subs="attributes,verbatim,macros,specialchars,replacements"]
+----
+psql -h 127.0.0.1 -d {PRODUCT_INIT}7 -U {PRODUCT_INIT} -W <pgdump.sql 
+----
++
+When prompted, enter the recorded password. The restoration can take considerable time. 
++
+Ignore database error messages. These messages are displayed because a database with the sane name already exists.
```

Comment:
I wouldn't say just _ignore_ any error message but something like _review the error messages and disregard those mentioning the object already exist_